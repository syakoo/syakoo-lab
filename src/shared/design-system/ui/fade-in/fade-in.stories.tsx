import type { Meta, StoryObj } from "@storybook/nextjs";

import { theme } from "@/shared/design-system/theme.css";

import { FadeIn } from ".";

const meta = {
  component: FadeIn,
  tags: ["autodocs"],
  parameters: { testLevel: "snapshot" },
} satisfies Meta<typeof FadeIn>;

export default meta;

const SampleInnerContent = () => (
  <div
    style={{
      borderRadius: theme.radius[100],
      background: theme.color.palette.gray[100],
      padding: "50px",
      textAlign: "center",
    }}
  >
    Children
  </div>
);

type Story = StoryObj<typeof meta>;
export const Sample: Story = {
  args: {
    children: <SampleInnerContent />,
  },
};
