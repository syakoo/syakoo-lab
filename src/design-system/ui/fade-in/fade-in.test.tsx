import { testStories } from "@/test/test-stories";

import * as stories from "./fade-in.stories";

describe("FadeIn", () => {
  testStories(stories);
});
