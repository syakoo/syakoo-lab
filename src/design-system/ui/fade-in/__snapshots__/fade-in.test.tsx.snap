// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FadeIn スナップショット Sample 1`] = `
<body>
  <div>
    <div
      class="fade-in_fadeInStyle__mal1bb1"
      style="animation-duration: 0.5s; animation-delay: 0s;"
    >
      <div
        style="border-radius: var(--radius-100__13jk9q11g); padding: 50px; text-align: center;"
      >
        Children
      </div>
    </div>
  </div>
</body>
`;
